### PR TITLE
Apply publint recommendations

### PR DESCRIPTION
**Pull request by dbritto-dev**
## Related Issues or Discussions

Publint warnings -> https://publint.dev/valtio@1.11.3

## Summary

Apply publint recommendations

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,13 +18,18 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "types": "./index.d.ts",
+      "module": {
```
Comment (dai-shi) [inline package.json:21]: "module" should have less priority than "import".

**Review by dai-shi — COMMENTED** [review]
Thanks for opening up a PR.

**Comment by dbritto-dev** [comment]
@dai-shi all yours

**Review by dai-shi — APPROVED** [review]
LGTM

**Pull request merged**